src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,49 @@
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import classNames from 'classnames';
+import { useContext } from 'react';
+import { PeopleContex } from '../../peopleContext';
+
+type Props = {
+  person: Person;
+};
+
+const getPersonByName = (
+  name: string | null,
+  people: Person[],
+): Person | undefined => {
+  return people.find(person => person.name === name);
+};
+
+export const PersonItem: React.FC<Props> = ({ person }) => {
+  const { slug: userId, sex, born, died, fatherName, motherName } = person;
+  const { slug: selectedUserId } = useParams();
+
+  const people = useContext(PeopleContex);
+  const mother = getPersonByName(motherName, people);
+  const father = getPersonByName(fatherName, people);
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({
+        'has-background-warning': userId === selectedUserId,
```

Comment:
The use of `classNames` library is great to conditionally apply styles. Good job!